webapp/src/test/java/rocks/metaldetector/web/controller/rest/ReleasesRestControllerIT.java

```diff
@@ -111,4 +124,16 @@ void user_not_is_allowed_to_update_release_state() throws Exception {
           .andExpect(status().isForbidden());
     }
   }
+
+  @Nested
+  class AnonymousRoleTest {
+
+    @Test
+    @DisplayName("Anonymous user is allowed to GET on endpoint " + TOP_UPCOMING_RELEASES + "'")
+    @WithMockUser(roles = "ANONYMOUS")
```

Comment:
```suggestion
    @WithAnonymousUser
```
See comment in `ArtistsRestControllerIT`